incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -315,6 +316,19 @@ whichever labels they like. However, HNC will overwrite any changes made to
 these labels, so other applications can trust these labels for policy
 application.
 
+<a name="basic-exceptions"/>
+
+### Exceptions and propagation control
+
+By default, HNC propagates all objects of a [given type](#admin-hc) from ancestor namespaces to descendant namespaces. However, sometimes this is too restrictive, and you need to create exceptions to certain policies. For example:
```

Comment:
Why "given type" is linked to hc?

Reply:
Oh good call. This should be a link to something about HNCConfig, not HierarchyConfig. There doesn't seem to be a good candidate in the concepts doc, so maybe linking to how-to.md#admin-resources would make more sense.

Reply:
Also instead of "given type," "specified type" probably would make more sense along with the link.